addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -23,10 +23,13 @@
                 <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_name">
                     <t t-esc="attachmentViewer.attachment.displayName"/>
                 </div>
+                <div class="o-autogrow"/>
                 <div class="o_AttachmentViewer_buttonDownload o_AttachmentViewer_headerItem o_AttachmentViewer_headerItemButton" t-on-click="_onClickDownload" role="button" title="Download">
```

Comment:
I still need to review old code part, but just a small remarks for the buttons here, it would be nice if the clickable area was the whole height, instead of being limited to icon size.

Reply:
Done